tests/e2e/suite_test.go

```diff
@@ -304,29 +305,52 @@ func DownloadBuiltISO(ns, seedName, filename string) {
 	// Set minimal ISO file to 250MB
 	const minimalISOSize = 250 * 1024 * 1024
 
-	// Check that the seed image is correctly created
-	Eventually(func() string {
-		out, _ := kubectl.RunWithoutErr("get", "SeedImage",
+	By("Waiting for image to be generated", func() {
+		// Check that the seed image is correctly created
+		Eventually(func() string {
+			out, _ := kubectl.RunWithoutErr("get", "SeedImage",
```

Comment:
Just nitpicking, but it would be best to test the seedimage builder pod readiness, so we can validate that's correct as well: `kubectl wait --for=condition=ready pod -n fleet-default fire-img`.
This test is also fine and equivalent however.

Reply:
Thanks, I will test this for another PR.